README.rst

```diff
@@ -21,151 +21,62 @@ Readme pyndl
-machine. If you only want to use ``pyndl`` as a python package use ``pip3`` in
-order to install it into your python3 environment. If you want to inspect and
-change the code download and install it via ``git clone`` and ``python3
-setup.py``. For details see below.
-
-
-Prerequisites
--------------
-You need python 3.5 or newer and git installed on your machine. We recommend to
-install Minicoda (https://conda.io/miniconda.html) before installing ``pyndl``
-or to create a virtualenv within your personal folder.
-
-Development
-^^^^^^^^^^^
-If you want to develop ``pyndl`` you should additionally install:
-
-.. code:: bash
-
-   pip3 install --user tox pylint pytest pycodestyle sphinx
-
-
-Installing
-----------
-If you only want to use the package run you can install ``pyndl`` from pypi with:
-
-.. code:: bash
-
-    pip3 install --user pyndl
-
-If you want to inspect and change the source code as well as running tests and
-having local documentation, clone the repository and install the package in
-'development' mode by running
-
-.. code:: bash
-
-    git clone https://github.com/quantling/pyndl.git
-    cd pyndl
-    python3 setup.py develop --user
-
-
-Documentation and Examples
---------------------------
-Documentation and examples can be found under
-http://pyndl.readthedocs.io/en/latest/ or in the ``doc/`` folder after cloning
-the repository.
-
-If you have installed ``sphinx`` you should be able to build the documentation with:
-
-.. code:: bash
-
-   cd doc/
-   make html
-
-The entry point for the html documentation is ``doc/build/html/index.html``.
-
-
-Running the tests
-=================
-If you have cloned ``pyndl`` from github, you can start the tests by executing
-the following within the outer ``pyndl`` folder:
-
-.. code:: bash
-
-    py.test
-    py.test doc/source/examples.rst
-
-
-For full tests you can run:
-
-.. code:: bash
 
-    tox -e test
+Installation
+============
 
-For manually checking coding guidelines run:
+The easiest way to install *pyndl* is using
+`pip <https://pip.pypa.io/en/stable/>`_:
 
 .. code:: bash
 
-    pycodestyle pyndl tests
-    pylint --ignore-patterns='.*\.so' --rcfile=setup.cfg -j 2 pyndl tests
-
-For more details on which tests run in the continuous testing environment
-look at the file ``tox.ini``.
+    pip install pyndl --user
 
+For more information have a look at the `Instalation Guide
+<http://pyndl.readthedocs.io/en/latest/installation.html>`_.
 
-Deployment
-==========
-In order to create a source dist package run:
 
-.. code:: bash
+Documentation
+=============
 
-    python3 setup.py sdist
+*pyndl* uses ``sphinx`` to create a documentation manual. The documentation is
+hosted on `Read the Docs <http://pyndl.readthedocs.io/en/latest/>`_.
 
 
-Contributing
-============
-Please read
-`CONTRIBUTING.rst
-<https://github.com/quantling/pyndl/blob/master/CONTRIBUTING.rst>`_ for details
-on our code of conduct and the process for submitting pull requests to us.
-
+Getting involved
```

Comment:
We definitly have to write something about the license here. If people contribute via pull request, they have to release their code under our provided license.

Reply:
I am not sure about that.

@derNarr what do you think?

Reply:
We added a license in the documentation's development section and in the PR template